crates/spfs/src/runtime/storage.rs

```diff
@@ -408,12 +408,12 @@ pub struct Config {
 
 impl Default for Config {
     fn default() -> Self {
-        Self::from_root(Path::new(Self::RUNTIME_DIR))
+        // Default RUNTIME_DIR
+        Self::from_root(temp_dir().join("spfs-runtime"))
```

Comment:
```suggestion
        let default_runtime_dir = temp_dir().join("spfs-runtime");
        Self::from_root(default_runtime_dir)
```

A named variable allows us to avoid the declarative comment :)